templates/bundles/TwigBundle/Exception/error404.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html.twig' %}
+
+
+{% block body %}
+
+	<div class="container p-4">
+
+		<p class="col text-center black-text">
+		    <h1>Oups 404</h1>
+
+			<h2>Notre application a retourné une erreur 404 : page inexistante.</h2>
+
+			<br>
+			<ul class="nav justify-content-center">
+				<li class="nav-item">
+					<a class="nav-link btn btn-sm btn-warning" href="{{ path("static") }}">Retour à la page d'accueil</a>
+
```

Comment:
Peux être voir une meilleur mise en forme du button retour à l'accueil ? Qu'est-ce que tu en pense ?

Reply:
Bien noté, je travaille sur ça rapidement